README.md

```diff
@@ -14,5 +14,12 @@
 * If dependencies have changed you may need to rebuild the docker images using `docker compose build`
 * To run an individual app rather than all of them, run `docker compose up appname` where app name is the key defined under `services` in [docker-compose.yml](docker-compose.yml) 
 * If you get an error about a database not existing, try running `docker compose down` followed by `docker compose up` this will remove and re-create any existing containers and volumes allowing the new databases to be created.
+* If file upload is not working with an error about credentials, you need to uncomment lines 128 and 129 of `docker-compose.yml` and put the credentials in your `.env` file. The credentials are stored in the DLUHC BitWarden vault.
```

Comment:
I guess those line numbers might change, can we link to a permalink of a revision instead e.g. https://github.com/communitiesuk/funding-service-design-docker-runner/blob/d13af481818fbd6398c3583e49a33edd6fb19496/docker-compose.yml#L114-L116 ?